### PR TITLE
Minor changes.

### PR DESCRIPTION
**Pull request by allelos**
- Increased specificity of Anchor styles
- Changed button label
- Changed copy under report a bug button

### CODE REVIEW

File: src/organisms/documentation/general.js [hunk 1]
```diff
@@ -35,20 +39,20 @@ const propsByApp = {
...
     documentationUrl: "https://learn.netdata.cloud/",
     issuesUrl: "https://github.com/netdata/netdata-cloud/issues/new",
-    issuesLabel: "Raise a bug about Netdata Cloud",
+    issuesLabel: "Raise a bug about Netdata Cloud or inform us for any issue you face.",
     otherIssuesUrl: "https://github.com/netdata/netdata/issues/new",
-    otherIssuesLabel: "an Agent",
+    otherIssuesLabel: "Agent",
     communityUrl: "https://community.netdata.cloud/c/support/13",
-    communitySupportUrl: "https://community.netdata.cloud/c/support/13/none",
+    communitySupportUrl: " https://www.netdata.cloud/community/",
```
Comment (apostolidhs) [inline src/organisms/documentation/general.js:46]: There is an extra space here

File: src/organisms/documentation/general.js [hunk 2]
```diff
@@ -6,11 +6,15 @@ import { Icon } from "src/components/icon"
...
 import Flex from "src/components/templates/flex"
 
-const Anchor = styled(Text).attrs({ as: "a", color: "primary", target: "_blank" })`
-  text-decoration: none;
-  &:hover {
-    ${getColor("accent")}
+const Anchor = styled(Text).attrs({ as: "a", target: "_blank" })`
+  &&& {
     text-decoration: none;
+    color: ${getColor("primary")};
```
Comment (apostolidhs) [inline src/organisms/documentation/general.js:12]: Text has already a color property

**Review by apostolidhs — APPROVED** [review]

File: src/organisms/documentation/general.js (hunk 2 above)
Comment (allelos) [reply src/organisms/documentation/general.js:12]: We need higher specificity here.

File: src/organisms/documentation/general.js (hunk 1 above)
Comment (allelos) [reply src/organisms/documentation/general.js:46]: Oops! I thought I pushed those typo fixes!

**Pull request merged**